iOS Client/PodcastsController/PodcastDetailViewController.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  PodcastDetailViewController.swift
+//  iOS Client
+//
+//  Created by Jonathan Witten on 12/21/17.
+//  Copyright © 2017 Mockingbird. All rights reserved.
+//
+
+import UIKit
+import ReSwift
+import AVFoundation
+
+class PodcastDetailViewController: UIViewController, StoreSubscriber {
+    
+    var podcast: MBPodcast?
+    var player: AVPlayer?
+    
+    @IBOutlet weak var titleLabel: UILabel!
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureBackButton()
+
+        // Do any additional setup after loading the view.
+    }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        MBStore.sharedStore.subscribe(self)
+    }
+    
+    override func viewWillDisappear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        MBStore.sharedStore.unsubscribe(self)
+    }
+    
+    func configureBackButton() {
+        self.navigationItem.leftBarButtonItem = UIBarButtonItem(title: "Back", style: .done, target: self, action: #selector(self.backToPodcasts(sender:)))
+    }
+    
+    @objc func backToPodcasts(sender: AnyObject) {
+        player = nil
```

Comment:
Why set player to nil? Won't the PopCurrentNavigation action deallocate this view controller and, in turn, its fields? Is there a noticeable delay prompting this?

Reply:
yeah probably don't need to do this at all